.github/workflows/build-binaries.yml

```diff
@@ -24,6 +23,7 @@ jobs:
                       target: node20-linux
                     - os: win
                       target: node20-win
+        runs-on: ${{ (matrix.arch == 'arm64' && matrix.os == 'linux') && 'ubuntu24-arm64' || 'ubuntu-20.04' }}
```

Comment:
Why this specific version of ubuntu? Can it just be `matrix.arch == 'arm64' && matrix.os == 'linux'`?